x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_instances/agent_contextual_information/index.tsx

```diff
@@ -40,6 +41,13 @@ const instancesLabel = i18n.translate(
   }
 );
 
+const latestVersionLabel = i18n.translate(
+  'xpack.apm.agentInstancesDetails.latestVersionLabel',
+  {
+    defaultMessage: 'Agent latest version',
```

Comment:
What about Latest version of agent? Or Latest agent version?

Reply:
Addressed in [3130a75](https://github.com/elastic/kibana/pull/153643/commits/3130a7506f9572fe9a633ffaeeacbf7c9b9937aa).